### PR TITLE
Verification

### PR DESCRIPTION
**Pull request by tamilan1998**

### CODE REVIEW

File: backend/models/search/UserSearch.php [hunk 1]
```diff
@@ -12,12 +12,19 @@
...
     public $branchId;
     public $fullName;
     public $dateOfJoining;
     public $fromDate;
     public $toDate;
+     public $pan;
+    public $aadhaar;
+    public $dateOfBirth;
+    public $personalPhone;
+ public $email;
```
Comment (MahaLakshmiMurugan) [inline backend/models/search/UserSearch.php]: no need to define a public property for email field

File: backend/models/search/UserSearch.php [hunk 2]
```diff
@@ -29,7 +36,7 @@ public function rules()
             [['id', 'status', 'created_at', 'updated_at', 'logged_at', 'branchId'], 'integer'],
             [['username', 'auth_key', 'password_hash', 'email', 'role_name'], 'safe'],
             [['fromDate'], 'date', 'format' => 'php:M Y'],
-            [['toDate'], 'safe'],
+            [['toDate','pan','aadhaar','dateOfBirth','personalPhone','email',], 'safe'],
```
Comment (MahaLakshmiMurugan) [inline backend/models/search/UserSearch.php]: kindly check https://github.com/senguttuvang/hrms/pull/16/files#diff-1492ec245f5312e3ac7c8cbad72a1de6R37 line

File: backend/views/layouts/common.php [hunk 3]
```diff
@@ -162,6 +162,11 @@
                     'label' => Yii::t('backend', 'Points calculator'),
                     'url' => ['/points/index'],
                     'visible' => Yii::$app->user->can('requirement.menu.access'),
+                ],
+                 [
+                    'label' => Yii::t('backend', 'candidate-verification '),
```
Comment (MahaLakshmiMurugan) [inline backend/views/layouts/common.php]: rename the label  candidate-verification  from Candidate Verification

Comment (tamilan1998) [reply backend/views/layouts/common.php]: image

File: backend/models/search/UserSearch.php (hunk 2 above)
Comment (tamilan1998) [reply backend/models/search/UserSearch.php]: image

**Pull request closed without merging**